packages/state-transition/src/block/processExecutionLayerConsolidationRequest.ts

```diff
@@ -0,0 +1,73 @@
+import {toHexString} from "@chainsafe/ssz";
+import {electra, ssz} from "@lodestar/types";
+import {FAR_FUTURE_EPOCH, MIN_ACTIVATION_BALANCE, PENDING_CONSOLIDATIONS_LIMIT} from "@lodestar/params";
+
+import {CachedBeaconStateElectra} from "../types.js";
+import {getConsolidationChurnLimit, isActiveValidator} from "../util/validator.js";
+import {hasExecutionWithdrawalCredential} from "../util/electra.js";
+import {computeConsolidationEpochAndUpdateChurn} from "../util/epoch.js";
+
+export function processExecutionLayerConsolidationRequest(
```

Comment:
the latest version of the spec is `process_consolidation_request()` so maybe we should go with that (without `ExecutionLayer`)